contracts/pre-propose/cwd-pre-propose-multiple/Cargo.toml

```
@@ -0,0 +1,39 @@
+[package]
+name = "cwd-pre-propose-multiple"
+version = "0.1.0"
+authors = ["ekez <zekemedley@gmail.com>", "Jake Hartnell <meow@no-reply.com>", "blue-note"]
+edition = "2018"
```

```suggestion
name = "cwd-pre-propose-multiple"
version = "0.2.0"
authors = ["ekez <zekemedley@gmail.com>", "Jake Hartnell <meow@no-reply.com>", "blue-note"]
edition = "2021"
repository = "https://github.com/DA0-DA0/dao-contracts"
description = "A DAO DAO pre-propose module for cwd-proposal-multiple for native and cw20 deposits."
```

moved us up to 2021 as part of the renaming changes. sorry for forgetting to mention. also the description and repo fields are nice to have once we publish to crates.io :)